raw/map.go

```diff
@@ -54,6 +54,10 @@ func (s *StringSliceMap) UnmarshalJSON(b []byte) error {
 			// split each slice element into key/value pairs
 			kvPair := strings.SplitN(v, "=", 2)
 
+			if len(kvPair) != 2 {
```

Comment:
could add something along the lines of https://github.com/go-vela/types/blob/8396842ec2d0693ae1f97f5372debf9577426117/yaml/ulimit.go#L62 here to silence the linter. i don't think we need to make it a constant.